tests/fixtures/Activity.json

```diff
@@ -0,0 +1,12 @@
+[
+{
+    "model": "registry.activity",
+    "pk": "c93134c8-123f-4ede-ada5-db8fe083fad5",
+    "fields": {
+        "name": "international",
```

Comment:
Suggest have a name like: photography, videotaping etc.

Reply:
Ah okay, I was actually using `faker` module to generate random words/ data for test database, Will update the fixtures accordingly.

Reply:
I case now we have a fixture data file. I think it should be possible to use that itself for test database. I'll check it once and make the changes accordingly

Reply:
Done in 2ef8fa8c3fcff67b6cafd105b39529cf5107983c